pkg/koordlet/runtimehooks/config_test.go

```diff
@@ -36,6 +36,11 @@ func Test_NewDefaultConfig(t *testing.T) {
 		RuntimeHookHostEndpoint:         "/var/run/koordlet/koordlet.sock",
 		RuntimeHookDisableStages:        []string{},
 		RuntimeHooksNRI:                 true,
+		RuntimeHooksNRIConnectTimeout:   2 * time.Second,
+		RuntimeHooksNRIBackOffDuration:  1 * time.Second,
+		RuntimeHooksNRIBackOffCap:       10 * time.Minute,
```

Comment:
should unlimitedly retry by default

Reply:
Sure, let's make nri reconnect unlimitedly.